### PR TITLE
Spelling and grammar

### PR DESCRIPTION
**Pull request by Giftlin**
img_20170915_230527
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/compat/chainmap_impl.py [hunk 1]
```diff
@@ -34,10 +34,10 @@ def wrapper(self):
 
 class ChainMap(MutableMapping):
     """ A ChainMap groups multiple dicts (or other mappings) together
-    to create a single, updateable view.
+    to create a single, updatable view.
 
     The underlying mappings are stored in a list.  That list is public and can
-    accessed or updated using the *maps* attribute.  There is no other state.
```
Comment (gfyoung) [inline pandas/compat/chainmap_impl.py]: Keep the space after the period.

Comment (gfyoung) [reply pandas/compat/chainmap_impl.py]: I see that it's because of `flake8` issue.  Perhaps replace the "or" with a slash to get it to fit. :smile:

Comment (Giftlin) [reply pandas/compat/chainmap_impl.py]: Are 2 whitespaces necessary? It had two, reduced to one.

Comment (gfyoung) [reply pandas/compat/chainmap_impl.py]: For consistency, yes.

File: pandas/compat/chainmap_impl.py [hunk 2]
```diff
@@ -35,9 +35,9 @@ def wrapper(self):
 class ChainMap(MutableMapping):
     """ A ChainMap groups multiple dicts (or other mappings) together
     to create a single, updatable view.
-
+
```
Comment (gfyoung) [inline pandas/compat/chainmap_impl.py]: Did you add a whitespace by accident?

File: pandas/compat/chainmap_impl.py (hunk 1 above)
Comment (Giftlin) [reply pandas/compat/chainmap_impl.py]: Can the slash have spaces before and after?

**Review by gfyoung — APPROVED** [review]
Ping when green and will merge!

File: pandas/compat/chainmap_impl.py (hunk 1 above)
Comment (gfyoung) [reply pandas/compat/chainmap_impl.py]: Perfectly fine.

**Comment by Giftlin** [comment]
@gfyoung 
img_20170916_074116
I don't see any green yet :(

**Comment by gfyoung** [comment]
Don't worry about it.  Yellow means the tests are still running.  You should only get concerned if it turns red.

**Comment by Giftlin** [comment]
@gfyoung all checks have passed.

**Comment by gfyoung** [comment]
Thanks @Giftlin !

**Pull request merged**